### DISCUSSION TITLE
Wireguard on DSM7.2 with Synology DS224+ doesn't work.

### DISCUSSION THREAD

**Discussion by Sha1key** [runfalk/synology-wireguard#186] [Q&A]
I'm installed the right package (gemini lake) but it doesn't work.
Uninstalled and reinstalled didn't work aswell? Any ideas?

**Comment by ingmarschmidt** [comment]
Get the pre-compiled package for DS224+ DSM7.2++ here: https://drive.google.com/drive/folders/1bY1u32nXS0Sj6tKfdZmuYon0sqFGoJpw
